tasks/build/build.js

```diff
@@ -57,10 +59,32 @@ var lessTask = function () {
     buildLess('app/stylesheets/builtin-pages.less', srcDir.path('stylesheets'))
   ])
 };
+
+var createLocalesFolder = function () {
+  console.log('os.platform', os.platform())
+  if (os.platform() !== 'darwin') {
+    return;
+  }
+  var options = {
+    continueOnError: false,
+    pipeStdout: false,
+    customTemplatingThing: "test"
+  };
+  var reportOptions = {
+    err: true,
+    stderr: true,
+    stdout: true
+  }
+  return gulp.src('./')
+    .pipe(exec('mkdir -p ./app/node_modules/locales/ && echo "{}" > ./app/node_modules/locales/en.json', options))
```

Comment:
I assume this command works on Mac, I tried it on Linux but it writes "{}" onto `en.json`.